tests/assets/schema_input.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I believe the tests should use `assets/schema_input.json`, so I'm wondering if this was left over from trying to get the tests to work? Could you remove this file if so?

Reply:
Yes! Good call.